src/components/proposalBooking/ProposalBookingDialog.tsx

```diff
@@ -21,112 +21,96 @@ import BookingEventStep from './bookingSteps/BookingEventStep';
 import ClosedStep from './bookingSteps/ClosedStep';
 import EquipmentBookingStep from './bookingSteps/EquipmentBookingStep';
 import FinalizeStep from './bookingSteps/FinalizeStep';
-import RequestReviewStep from './bookingSteps/RequestReviewStep';
-import ReviewFeedbackStep from './bookingSteps/ReviewFeedbackStep';
+// import RequestReviewStep from './bookingSteps/RequestReviewStep';
+// import ReviewFeedbackStep from './bookingSteps/ReviewFeedbackStep';
 
 const useStyles = makeStyles(() => ({
   fullHeight: {
     height: '100%',
   },
 }));
 
+export enum ProposalBookingSteps {
+  BOOK_EVENTS = 0,
+  BOOK_EQUIPMENT,
+  // REQUEST_REVIEW,
```

Comment:
Do we need to keep these comments? Is there something unfinished?

Reply:
Those are the unimplemented steps, I was uncertain to remove them or keep them as a comment.